conf.py

```diff
@@ -87,7 +87,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+#html_static_path = ['_static']
```

Comment:
Is commenting this out intentional ?

Reply:
Yes. Since the _static directory doesn't currently exist, the build pops a warning. I removed it to keep the build clean.